crossteam-project-valuation.md

```diff
@@ -0,0 +1,101 @@
+# Title
+
+Cross-team Project Valuation
+
+# Patlet
+
+It's hard to sell the value of cross-team, inner sourced projects that don't provide a direct impact on company revenue.
+Here's a data-driven way to represent your project that both articulates its value and amplifies it.
+
+# Context
+
+* You are responsible for a cross-team that serves as a platform for others in the company.
+* The cross-team project does not deliver any direct value for company revenue.
+
+# Problem
+
+Cross-team projects can potentially have a very large impact on the company yet are notoriosly-difficult to represent in a data-driven fashion.
```

Comment:
It would be helpful to explain why that is so notoriously difficult, @rrrutledge. It's not obvious to me ;)

Reply:
Removed the strong word, _notorious_, and moved this discussion to **Forces**.